### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by KhoiLe89**
Attended to the following "fixes:"

- Make sure all pages are responsive

- Added default view when user is not signed up to any presentations(hard coded for now)

- Added delete button to presentation#show and presentation#edit

- Added First and Last name to User Profile Edit page

Also, commented out presentations seeds to test the default response to "no presentations"

### CODE REVIEW

File: app/views/presentations/_no_presentations.html.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<h1>Attendee</h1>
```
Comment (domarp-j) [inline app/views/presentations/_no_presentations.html.erb]: I'd recommend not creating a new view & route for the 'no presentations' case. Instead, I'd add something like this to a view (or a view helper, to minimize logic within the view).

File: config/routes.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   # Root path
   root 'static#home'
-
+  get '/nopresentations', to: 'presentations#no_presentations'
```
Comment (domarp-j) [inline config/routes.rb]: Again, I'd try and take this out & work with the routes we have to show no presentations.

File: db/seeds.rb [hunk 3]
```diff
@@ -59,30 +59,30 @@
...
-      ques = survey.questions.create(
-        order: ques_num,
-        prompt: Faker::Hipster.words(5).join(' ') + '?',
-        response_type: ['text', 'number'].sample
-      )
-      ques.save
-    end
-  end
-end
+# puts "Creating presentations, surveys, and questions..."
```
Comment (domarp-j) [inline db/seeds.rb]: Why comment out this portion of the seeds file?

Comment (KhoiLe89) [inline db/seeds.rb]: To mimic users who have not signed up for presentations

**Pull request closed without merging**